### DISCUSSION TITLE
What do the "P" and "D" in the Flag column of the parameter table of the mitsuba document stand for?

### DISCUSSION THREAD

**Discussion by satoshiSchubert** [mitsuba-renderer/mitsuba3#645] [Q&A] [answered]
## Summary
What do the "P" and "D" in the Flag column of the parameter table of the mitsuba document stand for?

## System configuration

System information:

  Mitsuba version: mitsuba3

## Description
When reading the official mitsuba documentation, the FLAG column of the parameter table may have P, ∂, D flags. I know that ∂ means differentiable, but I don't know what P and D represent respectively. They may have been explained somewhere in the documentation, but unfortunately I haven't found it yet, hoping for a simple explanation

**Comment by njroussel** [comment, marked as answer]
Hi @satoshiSchubert 

Hovering over them should give you a tooltip explaining what they are used for. They're is also a small description [here](https://mitsuba.readthedocs.io/en/stable/src/plugin_reference.html#flags).

Basically, `P` means that they are exposed/modifiable through `mi.traverse()`. `D` is used to indicate that the parameter might introduce discontinuities when differentiating. And as you correctly mentionned already, `∂` is used to indicate whether or not the parameter is differentiable.

**Reply by satoshiSchubert** [reply]
Thanks for your quick reply!
